CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 Scalyr Agent 2 Changes By Release
 =================================
 
+## 2.1.7 "Serenity" - June 24, 2020
+
+<!---
+Packaged by Yan Shnayder <yan@scalyr.com> on Jun 24, 2020 16:30 -0800
+--->
+
+Features:
+* New configuration option `k8s_logs` allows configuring of Kubernetes logs similarly to the `logs` configuration option. Please see the [RELEASE_NOTES](https://github.com/scalyr/scalyr-agent-2/blob/master/RELEASE_NOTES.md) for more details.
```

Comment:
Also say "configuration but matches based on Kubernetes pod, namespace, and container name" instead of "configuration option" after `logs`.